Makefile

```diff
@@ -171,10 +171,21 @@ extension-test-build:
 		-DENABLE_ADDRESS_SANITIZER=TRUE \
 	)
 
+extension-json-test-build:
```

Comment:
Why not reuse the existing test command?
Do we need this extra command?

Reply:
The existing test command requires that people working on JSON also build DuckDB.